### DISCUSSION TITLE
How much can a modder physically customize a character in BombSquad at the highest level ever achieved?

### DISCUSSION THREAD

**Discussion by HeyErfan** [bombsquad-community/plugin-manager#309] [Q&A]
**How much can a modder physically customize a character in Bombsquad at the highest level ever achieved?**

If this question is too broad, let me start with: Can a modder achieve this?:
When a player presses the yellow button twice and the jump button once in quick succession (within about2 seconds), the character then moves forward in a certain scale, stops, punches twice, and returns to the exact coordinates related to the last button press (in this example, the jump button).

To make it a bit more complex: Is it possible to save a sequence of button presses and then apply it to a player’s character? This means I consider the standing position as the starting point and zero. We need the character to walk at least20 steps in a normal environment without hitting any obstacles (the player must position their character under these **conditions),** We then tell the player: “You just need to press the blue button twice and the yellow button once in quick succession. Now, we will automatically apply the bomb jump to your character” (this process requires someone who, while standing, starts running and performs a bomb jump, then saves the buttons and movements that lead to a successful bomb jump).
Can we utilize the stored movements in the form of a mod or something else?

Let me ask more questions:
Can we take the jump button out of its normal function? We could tell the player that after using this version of the jump, when they press the jump button again, they won't just see a normal jump. They will see the character perform a relatively high jump, spin around, and then return to where they jumped from.

Finally, I'm just trying to understand maybe complex and interesting relationships in Bombsquad. Why? It's fascinating :)

**Comment by brostosjoined** [comment]
Yes its possible

**Comment by brostosjoined** [comment]
The plugin made by @imayushsaini  "auto_stunt.py" does this already. You can check that out its available on plugman(plugin manager), then give us your feedback.

**Reply by HeyErfan** [reply]
**auto stunt** (by @vishal332008):
https://github.com/bombsquad-community/plugin-manager/blob/main/plugins/utilities/auto_stunt.py

**auto run** (by @TheMikirog ):
https://github.com/bombsquad-community/plugin-manager/blob/main/plugins/utilities/autorun.py

**Reply by Loup-Garou911XD** [reply]
> **auto run** (by @Loup-Garou911XD):
> https://github.com/bombsquad-community/plugin-manager/blob/main/plugins/utilities/autorun.py

Autorun is by @TheMikirog

**Reply by HeyErfan** [reply]
Thanks, edited.
